tests/npbench/weather_stencils/vadv_test.py

```diff
@@ -211,7 +211,9 @@ def run_vadv(device_type: dace.dtypes.DeviceType):
     return sdfg
 
 
-def test_cpu():
+def test_cpu(monkeypatch):
+    # NOTE: Serialization fails because of "k - k" expression simplified to "0"
+    monkeypatch.setenv("DACE_testing_serialization", 0)
```

Comment:
Did we not disable serialization in tests altogether?

Reply:
It looks like we didn't make the PR yet.